lambdas/status_reports/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="t4_lambda_status_reports",
+    version="0.0.1",
+    packages=find_packages(where="src"),
+    package_dir={"": "src"},
+    install_requires=[
+        "Jinja2 ==3.1.2",
+        "aiobotocore ==2.3.4",
+        "botocore ==1.24.21",
```

Comment:
I think either of these are far more common:
```suggestion
        "Jinja2==3.1.2",
        "aiobotocore==2.3.4",
        "botocore==1.24.21",
```
```suggestion
        "Jinja2 == 3.1.2",
        "aiobotocore == 2.3.4",
        "botocore == 1.24.21",
```

Reply:
ok, i just saw this style on some docs page and considered it "official" 🤷

Reply:
> ok, i just saw this style on some docs page and considered it "official" shrug

I think I've never seen this before :man_shrugging:.